acceptance/bundle/deploy/experimental-python/script

```diff
@@ -0,0 +1,7 @@
+cd my_project
+
+trace uv run --quiet --python 3.12 --with databricks-bundles==0.7.0 -- $CLI bundle deploy
+
+trace $CLI jobs list --output json
+
+rm -rf .databricks __pycache__ .gitignore
```

Comment:
There is also Ignore setting in test.toml that you can use (and share between tests).